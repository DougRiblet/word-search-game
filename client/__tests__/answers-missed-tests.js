import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';
import AnswersMissed from '../components/Answers-Missed/answers-missed-present';

describe('Answers Missed', () => {
  test('Snapshot', () => {
    const dummystring = 'ABCD EFGH MNOP';
    const AnswersMissed_tree = renderer.create(
      <AnswersMissed poolfour={dummystring} showmissed={true} />
    ).toJSON();
    expect(AnswersMissed_tree).toMatchSnapshot();
  });

  test('renders without crashing', () => {
    const dummystring = 'ABCD EFGH MNOP';
    const div = document.createElement('div');
    ReactDOM.render(<AnswersMissed poolfour={dummystring} showmissed={true} />, div);
  });
});
