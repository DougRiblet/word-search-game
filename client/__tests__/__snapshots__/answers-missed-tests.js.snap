// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Answers Missed Snapshot 1`] = `
<div
  id="answers-missed"
>
  ABCD EFGH MNOP
</div>
`;
